Gible.Domain/Commands/IngestRecipesCommand.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Gible.Domain.Models;
+using Gible.Domain.Repositories;
+using Knox.Commanding;
+using Knox.Extensions;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace Gible.Domain.Commands
+{
+    public record IngestRecipesCommand(string InputDirectory, string OutputDirectory) : Command;
+    public class InjestRecipesCommandHandler(IRepository<Recipe> recipeRepository) : ICommandHandler<IngestRecipesCommand>
+    {
+        private record FileInformation(string PrimaryLocation, string RecipeTitle, string NewDirectory);
+        public Task<bool> CanExecuteAsync(IngestRecipesCommand command)
+        {
+            throw new NotImplementedException();
+        }
+
+        public async Task ExecuteAsync(IngestRecipesCommand command)
+        {
+            // Get the recipes based off the images in the ingest folder.
+            // Copy the images into the processed folder.
+            // Delete the original images.
+
+            throw new NotImplementedException("Command is not yet complete.");
```

Comment:
Since it's beyond the scope of the branch's purpose, I'm going to leave the command unfinished